File: use-cases/school-timetabling/src/main/java/org/acme/schooltimetabling/persistence/RoomRepository.java
package org.acme.schooltimetabling.persistence;

import jakarta.enterprise.context.ApplicationScoped;

import org.acme.schooltimetabling.domain.Room;

import io.quarkus.hibernate.orm.panache.PanacheRepository;

@ApplicationScoped
public class RoomRepository implements PanacheRepository<Room> {

}

File: technology/kotlin-quarkus/src/main/kotlin/org/acme/kotlin/schooltimetabling/persistence/RoomRepository.kt
package org.acme.kotlin.schooltimetabling.persistence

import io.quarkus.hibernate.orm.panache.kotlin.PanacheRepository
import org.acme.kotlin.schooltimetabling.domain.Room
import jakarta.enterprise.context.ApplicationScoped

@ApplicationScoped
class RoomRepository : PanacheRepository<Room>
